### PR TITLE
hooks: win32com: avoid printing ModuleNotFindError when module is unavailable

### PR DESCRIPTION
**Pull request by rokm**
Use a custom `exec_statement()` to query the `win32com.__file__` instead of using `get_module_file_attribute()`, as the latter
will print a `ModuleNotFindError` to stderr when the module is unavailable. The error message, while harmless, appears
to be bothering the users.

Fixes pyinstaller/pyinstaller#3733.

### CODE REVIEW

File: src/_pyinstaller_hooks_contrib/hooks/pre_safe_import_module/hook-win32com.py
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/hooks/pre_safe_import_module/hook-win32com.py:44]: Is there a reason you chose to keep the `get_module_file_attribute()` rather than using the output from the `exec_statement()` call?

Comment (rokm) [reply src/_pyinstaller_hooks_contrib/hooks/pre_safe_import_module/hook-win32com.py:44]: Ah, I didn't pay enough attention to what `get_module_file_attribute` actually does. But you're right, it is redundant now.

**Review by bwoodsend — APPROVED** [review]
That makes more sense :)

**Comment by bwoodsend** [comment]
I'll let @Legorooj merge this when he wake up.

**Review by Legorooj — APPROVED** [review]

**Comment by bwoodsend** [comment]
Appveyor is failing which is unfortunate because it's the one running Windows which we're trying to test. But the tests that are failing are `pyenchant` and `pylint` which I've seen fail elsewhere...

**Pull request merged**
